crates/contracts/src/vault.rs

```diff
@@ -45,32 +38,27 @@ pub async fn grant_required_roles(
 
 #[cfg(test)]
 mod tests {
-    use {super::*, ethcontract::H256};
+    use alloy::primitives::b256;
+
+    use super::*;
```

Comment:
This and other `use` statements should follow the convention that they go into one big block. Not sure why `cargo fmt` doesn't complain here. 🤔